Core/src/Utilities/AnnealingUtility.cpp

```diff
@@ -8,16 +8,20 @@
 
 #include "Acts/Utilities/AnnealingUtility.hpp"
 
+#include "Acts/Utilities/AlgebraHelpers.hpp"
```

Comment:
```suggestion
#include "Acts/Utilities/AlgebraHelpers.hpp"
#include "Acts/Utilities/AnnealingUtility.hpp"
```

Reply:
I think we (and clang-format) treat the *main* include (i.e. the one matching the `.cpp`) separately on it's own line.